src/NotificationServices/Softeq.NetKit.Services.SmsNotifications/SmsSender/ISmsSender.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System.Threading.Tasks;
+using Softeq.NetKit.Services.SmsNotifications.Models;
+
+namespace Softeq.NetKit.Services.SmsNotifications.SmsSender
+{
+    public interface ISmsSender
+    {
+        Task SendAsync(SendSmsDto sms);
```

Comment:
rename this dto. It looks strange without duplicating method name

Reply:
done